environment.yml

```diff
@@ -5,7 +5,7 @@ channels:
   - conda-forge
 dependencies:
   - biopython>=1.78
-  - kallisto>=0.46.1
+  - kallisto=0.48.0
```

Comment:
Still better to use a range. For example, >=0.46.1, <= 0.48.0

Reply:
Thanks, fixed in commit.